packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -260,6 +260,12 @@ export function getPortLabel({ label, type, isOptional }: WorkflowPort) {
 	return portLabel;
 }
 
+// Checker for resource-operators (e.g. model, dataset) that automatically create an output
+// without needing to "run" the operator because we can drag them onto the canvas
+export function canPropagateResource(outputs: WorkflowOutput<any>[]) {
+	return _.isEmpty(outputs) || (outputs.length === 1 && !outputs[0].value);
+}
```

Comment:
This pattern is used in non resource operators as well. The input is typically the first output we want to propagate. I use it in model-edit

Reply:
I don't see any append-output-port in `tera-model-edit-node`

Reply:
It's in the drilldown

Reply:
Gotcha, I would say that is kind of a special case then - that you would propagate when opening the drilldown. Will keep a note on it and revisit.